server/src/services/event/EventService.js

```diff
@@ -1,27 +1,62 @@
 const dayjs = require('dayjs');
 const timeSpans = require('./timeSpans');
 const scopes = require('./scopes');
+const AnalyzerService = require('./AnalyzerService');
+const ActionService = require('./ActionService');
 
 class EventService {
     constructor(prefix, redisService) {
         this.prefix = prefix;
         this.redisService = redisService;
+        this.actionService = new ActionService(prefix, redisService);
     }
 
     get stores() {
         const redisService = this.redisService;
+        const actionService = this.actionService;
 
         return [
             function (prefix, key, userId) {
```

Comment:
you can check arrow function, then "this" should be assigned to class automatically, but yeah, it's strange

Reply:
eventually you can always do:
const that = this;

Reply:
but of course, as you have it right now is also good